docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -49,6 +49,26 @@ To change the width of a column, use the `width` property available in `ColDef`.
 
 {{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
 
+## Column fluid width
+
+By default each column have a fixed width of 100 pixels but we can have fluid (responsive) columns by setting the `flex` propery avaibale in the `ColDef`.
```

Comment:
The usage of "we" is forbidden in the documentation (or at least limited to rare exceptions) https://trello.com/c/1Q7hqXHu/2714-technical-documentation-guidelines

Reply:
My bad, sorry about that, I'll fix it now.

Reply:
No problem! I'm doing this too from time to time. Matt is rectifying me when I do it wrong.